src/ex_docmd.c

```diff
@@ -460,6 +460,37 @@ restore_dbg_stuff(struct dbg_stuff *dsp)
 }
 #endif
 
+// @param fnum A buffer number. 0 == current buffer, 1-or-more must be a valid buffer ID.
+// @param ffname The full path to where a buffer lives on-disk or would live on-disk.
+// @return if `ffname` is the same as `fnum` buffer, return true.
+//
+static int is_other_file(int fnum, char_u *ffname)
```

Comment:
Style: please move `static int` to the previous line (with some indent) and start this line with `is_other_file`

Reply:
I think I got it this time, please let me know if not

Reply:
thanks!